projects/plugins/crm/src/automation/class-automation-engine.php

```diff
@@ -241,7 +241,7 @@ public function get_logger(): ?Automation_Logger {
 	}
 
 	/**
-	 * Get the registered triggers
+	 * Get the registered triggers.
```

Comment:
We are missing most of our `@return` tags.
In this case it would be something like:
```
 * @return array An associative array containing the registered triggers,
 *               where the keys represent the trigger names and the values represent
 *               their corresponding slugs.
```